cms/djangoapps/api/v1/serializers/course_runs.py

```diff
@@ -168,29 +168,33 @@ def create(self, validated_data):
 class CourseRunRerunSerializer(CourseRunSerializerCommonFieldsMixin, CourseRunTeamSerializerMixin,
                                serializers.Serializer):
     title = serializers.CharField(source='display_name', required=False)
+    number = serializers.CharField(source='id.course', required=False)
     run = serializers.CharField(source='id.run')
 
-    def validate_run(self, value):
+    def validate(self, attrs):
```

Comment:
Nit: I've seen this as `data` before, but I'm not sure if that's convention or not. I mention this only because we're starting to use the `attrs` lib more and more.

Reply:
There is a pylint error if it is set to `data`, W0221 I believe, which complains about the arguments differing from the function it is overwriting and it is `attrs` in `serializers.Serializer`. Made me sad too. I wanted to use `data`